airflow/providers/snowflake/transfers/copy_into_snowflake.py

```diff
@@ -0,0 +1,149 @@
+        (IdP) that has been defined for your account
+        ``https://<your_okta_account_name>.okta.com`` to authenticate
+        through native Okta.
+    :param session_parameters: You can set session-level parameters at
+        the time you connect to Snowflake
+
+    """
+
+    template_fields: Sequence[str] = ("files",)
+    template_fields_renderers = {"files": "json"}
+
+    def __init__(
+        self,
+        table: str,
+        file_format: str,
+        stage: str,
+        stage_postfix: Optional[str] = None,
+        columns_array: Optional[List[str]] = None,
+        files: Optional[List[str]] = None,
+        pattern: Optional[str] = None,
+        copy_into_postfix: Optional[str] = None,
+        snowflake_conn_id: str = "snowflake_default",
+        account: Optional[str] = None,
+        warehouse: Optional[str] = None,
+        database: Optional[str] = None,
+        region: Optional[str] = None,
+        role: Optional[str] = None,
+        schema: Optional[str] = None,
+        authenticator: Optional[str] = None,
+        session_parameters: Optional[dict] = None,
+        autocommit: bool = True,
+        *args,
+        **kwargs,
+    ):
+        super().__init__(*args, **kwargs)
+        self.table = table
+        self.file_format = file_format
+        self.stage = stage
+        self.stage_postfix = stage_postfix
+        self.columns_array = columns_array
+        self.files = files
+        self.pattern = pattern
+        self.copy_into_postfix = copy_into_postfix
+        self.snowflake_conn_id = snowflake_conn_id
+        self.account = account
+        self.warehouse = warehouse
+        self.database = database
+        self.region = region
+        self.role = role
+        self.schema = schema
+        self.authenticator = authenticator
+        self.session_parameters = session_parameters
+        self.autocommit = autocommit
+
+    def execute(self, context: Context) -> None:
+        snowflake_hook = SnowflakeHook(
+            snowflake_conn_id=self.snowflake_conn_id,
+            account=self.account,
+            warehouse=self.warehouse,
+            database=self.database,
+            region=self.region,
+            role=self.role,
+            schema=self.schema,
+            authenticator=self.authenticator,
+            session_parameters=self.session_parameters,
+        )
+
+        if self.schema:
+            into = f"{self.schema}.{self.table}"
+        else:
+            into = self.table
+
+        if self.columns_array:
+            into = f"{into}({', '.join(self.columns_array)})"
+
+        sql = f"""
+        COPY INTO {into}
+             FROM  @{self.stage}/{self.stage_postfix or ""}
+        {"FILES=" + ",".join(map(enclose_param ,self.files)) if self.files else ""}
+        {"PATTERN=" + enclose_param(self.pattern) if self.pattern else ""}
+        FILE_FORMAT={self.file_format}
+        {self.copy_into_postfix or ""}
+        """
```

Comment:
Is it really required to use a multi-line string syntax rather than prepare as string in parts, as it was done in `S3ToSnowflakeOperator` ?

Reply:
I rather to use multiline f-string, it's more clear of string block construction